conventions.md

```diff
@@ -1,3 +1,58 @@
 # Cryptic Conventions
-You can find the conventions in our official Wiki:
-- [Konventionen (Deutsch)](https://wiki.cryptic-game.net/books/einf%C3%BChrungen/page/konventionen)
```

Comment:
Please discuss if the DE conventions file needs to remain in the wiki - as most of the information is Github related, it might make sense to link from the wiki to GitHub instead....